examples/worldview/src/features/map/Map.js

```diff
@@ -40,13 +57,17 @@ export class Map extends Component {
 
     this._onMapLoad = this._onMapLoad.bind(this);
     this._resizeVideo = this._resizeVideo.bind(this);
+    this._resizeMap = this._resizeMap.bind(this);
+    this._changeDpi = this._changeDpi.bind(this);
+  }
 
+  componentDidMount() {
     this._resizeVideo();
   }
 
   componentDidUpdate(prevProps) {
-    const {dimension, previewSize} = this.props;
-    if (!isEqual(prevProps.dimension, dimension) || !isEqual(prevProps.previewSize, previewSize)) {
+    const {dimension} = this.props;
+    if (!isEqual(prevProps.dimension, dimension)) {
```

Comment:
Was able to remove updating when `previewSize` changed since the scaling is done with CSS outside of the GL context.